pkg/networkservice/connectioncontextkernel/ipcontext/ipaddress/client_test.go

```diff
@@ -37,13 +39,20 @@ import (
 	"github.com/networkservicemesh/sdk-vppagent/pkg/networkservice/vppagent"
 )
 
-const IPAddress = "1.1.1.1"
+const (
+	IPAddress    = "1.1.1.1"
+	netnsFileURL = "/proc/12/ns/net"
```

Comment:
Is this ok to hard code? no side effects in test?

Reply:
It has no side effects, because the tests just test the vpp-agent config generated... they don't attempt to apply them.